Gemfile

```diff
@@ -10,41 +10,8 @@ group :development, :test do
   gem "simplecov", require: false
 end
 
-# BEGIN ENGINE_CART BLOCK
```

Comment:
engine-cart auto-detects this block and adds it when it is missing, it does not have to be in the repo.